test/images/scripts/lib/debian.install

```diff
@@ -46,6 +46,9 @@ if [ -n "$do_build" ]; then
     tar -xzf cockpit-*.tar.gz
     ( cd cockpit-*/
       cp -rp tools/debian debian
+      # Hack: Remove PCP build dependencies while pcp is not in testing
+      # (https://tracker.debian.org/pcp)
+      sed -i '/libpcp.*-dev/d' debian/control
```

Comment:
To the contrary, this PR now would start to *allow* us doing weekly releases into unstable/testing as a new cockpit can actually propagate. It can't while it build/binary depends on pcp which isn't available in testing, and at the moment we can't even build cockpit with PCP in unstable.

The price is of course to drop the -pcp binary for the time being, but that's still better than dropping cockpit entirely IMHO.